.circleci/config.yml

```diff
@@ -218,13 +233,20 @@ workflows:
             - Github
 
   e2e:
-    when: << pipeline.parameters.e2e >>
+    when:
+      and:
+        - << pipeline.parameters.e2e >>
+        - not:
+          equal: [ clear_pnpm_cache, << pipeline.schedule.name >> ]
```

Comment:
does this need another intent?

Reply:
I don't think so, it passes the schema validation and didn't trigger on the daily run.

Reply:
interesting, yaml is confusing sometimes :)